meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -74,3 +74,4 @@ RESIN_FLASHER_FLAG_FILE = "resin-image-flasher"
 INITRAMFS_IMAGE = "resin-image-initramfs"
 INITRAMFS_IMAGE_BUNDLE = "1"
 KERNEL_INITRAMFS = "-initramfs"
+INITRAMFS_TASK = ""
```

Comment:
The INITRAMFS_TASK was used to define the task that was creating the initramfs, so the build system would create the proper dependencies. This was replaced by INITRAMFS_IMAGE.

    # NOTE: setting INITRAMFS_TASK is for backward compatibility
    #       The preferred method is to set INITRAMFS_IMAGE, because
    #       this INITRAMFS_TASK has circular dependency problems
    #       if the initramfs requires kernel modules
    image_task = d.getVar('INITRAMFS_TASK', True)
    if image_task:
        d.appendVarFlag('do_configure', 'depends', '${INITRAMFS_TASK}')
http://git.yoctoproject.org/cgit.cgi/poky/tree/meta/classes/kernel.bbclass?h=morty#n71